grails-app/services/org/pih/warehouse/core/LocationService.groovy

```diff
@@ -102,6 +102,11 @@ class LocationService {
 						(it.locationGroup != currentLocation.locationGroup && it.supports(ActivityCode.MANAGE_INVENTORY) && it.supports(ActivityCode.RECEIVE_STOCK)) }
 			}
 		}
+
+		if (params.locationTypeCode) {
+			return locations.findAll { it.locationType.locationTypeCode == LocationTypeCode.DEPOT }
```

Comment:
@DominikaHatala Quick question. It seems like this would make more sense as 
```
if (params.locationTypeCode) {
    LocationTypeCode locationTypeCode = params.locationTypeCode as LocationTypeCode
    return locations.findAll { it.locationType.locationTypeCode == locationTypeCode }
}
```

Reply:
yeah (assuming that we have locationTypeCode in params, otherwise this if does not make sense ;P )

Reply:
@jmiranda I added the fix